packages/medusa/src/utils/clean-response-data.ts

```diff
@@ -1,21 +1,53 @@
 import { pick } from "lodash"
 
+// TODO: once the legacy totals decoration will be removed.
+// We will be able to only compute the totals if one of the total fields is present
+// and therefore avoid totals computation if the user don't want them to appear in the response
+// and therefore the below const will be removed
+const EXLCUDED_FIELDS = [
```

Comment:
self-review: did not find a better name for explicitly saying that those props are excluded from being striped out

Reply:
typo: `EXCLUDED_FIELDS`

Reply:
ahah thanks @carlos-r-l-rodrigues